### ISSUE TITLE
Readme unclear, Does this work with minitest::unit?

### ISSUE THREAD

**Issue by hxegon** [mfpiccolo/minitest-vcr#23]
The readme implies that this gem only works for minitest::spec, is this the case?

If so, Do you think adding unit test support would be worth it?

**Comment by sheerun** [comment]
Same issue

**Comment by mfpiccolo** [comment]
Yes, as of right now this only works with spec.  I rarely use unit but I would be happy to add support for it if someone wants to get a spike or PR together.

**Comment by hxegon** [comment]
If no one has done it by, say, the end of August I'll see what I can do.
